packages/api-electron/package.json

```diff
@@ -2,14 +2,15 @@
   "name": "containerjs-api-electron",
   "version": "0.0.2",
   "description": "",
-  "main": "index.js",
+  "main": "build/lib/preload.js",
   "module": "build/es/preload.js",
   "jsnext:main": "build/es/preload.js",
   "browser": "build/dist/containerjs-api.js",
   "scripts": {
     "clean": "rimraf build",
-    "build": "npm run clean && npm run build:es && rollup -c",
-    "build:es": "rimraf build/es && tsc",
+    "build": "npm run clean && npm run build:cjs && npm run build:es && rollup -c",
+    "build:es": "rimraf build/es && tsc -p tsconfig.json -m ES2015 --outdir ./build/es",
```

Comment:
This change means that the user would no longer have access to Electron's main process when using this package; I think it makes sense to do this, as there's no equivalent in OpenFin/Browser.

 I've detailed the use of each entry point/build here: https://github.com/symphonyoss/ContainerJS/issues/132#issuecomment-302962997 - if this needs clarifying or changing, please let me know :)